commcare_connect/users/models.py

```diff
@@ -69,3 +69,15 @@ class Role(models.TextChoices):
         related_name="memberships",
     )
     role = models.CharField(max_length=20, choices=Role.choices, default=Role.MEMBER)
+
+
+class ConnectUser(BaseModel):
```

Comment:
As I mentioned on the ticket, I don't think these fields are quite right and we should hold off on adding this data until the details of ConnectID/Connect authentication are finished. As mentioned in the other comment, I don't think this is necessary for this PR either.

Reply:
Removed ConnectUser. [45f8e61](https://github.com/dimagi/commcare-connect/pull/41/commits/45f8e61ef0d11a964d253808d2ed907ef223b36b)